src/live_restore/live_restore_fs.c

```diff
@@ -664,12 +664,6 @@ __live_restore_fh_read(
     read_data = (char *)buf;
 
     __wt_readlock(session, &lr_fh->bitmap_lock);
-
-    /*
-     * FIXME-WT-13828: WiredTiger will read the metadata file after creation but before anything has
```

Comment:
As far as I can tell this is not relevant anymore as we refuse to do a live restore without a pre-existing database, originally we could always start in live restore mode.